src/Components/List/index.jsx

```diff
@@ -94,7 +94,7 @@ function List({ list, toggleComplete, deleteItem }) {
                           marginTop="sm"
```

Comment:
This comment is for line 77, but github won't let me comment there :(

In addition to updating the key to `item._id`, the key needs to be a property of the outermost component that your map renders. That means instead of fragments, we need to use another `<Container>` or similar so it can include the key property.